consultation_analyser/consultations/jinja2/schema.html

```diff
@@ -0,0 +1,123 @@
+{% extends "base.html" %}
+
+{% set page_title = "Consultation data schema" %}
+
+{% block content %}
+  <h1 class="govuk-heading-xl">Consultation data schema v0.1</h1>
+
+  <p class="govuk-body">This page lists the data components required for a consultation to be suitable for processing by the analyser. </p>
+
+  <p class="govuk-body">Data should be provided in one JSON file containing both the consultation itself and a list of responses. That overall data file is described here and an example is given. Underneath are details of each section, including individual JSON schemas for both the Consultation and ConsultationResponse parts of the JSON file.</p>
```

Comment:
I wasn't entirely clear on the difference between the 3 schemas and how they are related.

Could make it clear that the `ConsultationWithResponses` is the one we want e.g. "Data should be provided in one JSON file..... that is valid according to the ConsultationWithResponses schema."

Could be explicit on how the 3 schema relate.